plugins/timesync.koplugin/main.lua

```diff
@@ -19,6 +19,8 @@ local NetworkMgr = require("ui/network/manager")
 
 local TimeSync = WidgetContainer:new{
     name = "timesync",
+    fullname = _("Time sync"),
+    description = _([[Synchronizes the device time with NTP servers.]]),
```

Comment:
Synchronizes the device with with NTP servers to within a few milliseconds of UTC?

I don't know, that might still sound a bit mystical.

Reply:
Synchronizes the time between the device and the time on the Internet. ?